lib/ddtrace/contrib/racecar/patcher.rb

```diff
@@ -0,0 +1,33 @@
+module Datadog
+  module Contrib
+    module Racecar
+      module Patcher
+        include Base
+        register_as :racecar
+        option :service_name, default: 'racecar'
+
+        module_function
+
+        def patch
+          return patched? if patched? || !defined?(::Racecar)
+
+          require_relative 'tracer'
+
+          ::Racecar.singleton_class.class_eval do
+            alias_method :__instrumenter, :instrumenter
```

Comment:
Would it be possible to make this work now that the `start_` notifications are emitted?